Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,45 @@
+.tech {
+  background-color: $green;
+  border-radius: 5px;
+  padding: 48px;
+  position: relative;
+  width: 250px;
+  
+  &:before {
+    content: "";
+    border-radius: 5px;
+    border:1px solid $green;
+    height:100%;
+    left: -10px;
+    position: absolute;
+    top: -10px;
+    width: 100%;
+    z-index: -1;
+  }
+  
+  .img-tech {
+    height: $tech-logo-size;
+    margin: 0 0 10px 0;
```

Comment:
you don't need the 0 values since the equalizer removes all the margins. just say `margin-bottom: 10px;`